aerogear/mobile-developer-console
# AEROGEAR-8137

How to verify:
- Make sure you have the latest UPS apb. If not, change metadata of UPS apb to https://github.com/aerogearcatalog/unifiedpush-apb/blob/master/apb.yml#L32 via `oc edit clusterserviceplan`
- Start this project, Provision UPS, provision Metrics
- Create a client and bind to UPS
- You will still see the Bind button next to UPS, but not for e.g. Metrics when you bind to Metrics
- Press Bind button next to UPS again
- You will only see the missing platform, Android or IOS, in the platform select box
- Create the binding
- You won't see the Bind button next to UPS anymore
- Also test things like "Binding in Progress" status display, etc.

What's missing:
- Delete button adaption for multiple bindings. Will be done separately within a task from https://issues.jboss.org/browse/AEROGEAR-8153
